MotorizationRate.ipynb

```diff
@@ -148,7 +148,7 @@
     }
```

Comment:
<p>not PEP8 valid</p><p> </p><p>personal opinion, but comments must not follow a line of code but be on the line above</p><p> </p><pre class="ql-syntax" spellcheck="false">this_is()
a_bad_thing() # do a bad thing
</pre><p> </p><p>vs</p><p> </p><pre class="ql-syntax" spellcheck="false">this_is()
# make it right
a_good_thing()
</pre><p> </p><p>this enhances readability</p><p> </p><p>also, <code>cor_selector</code> is not an understandable name, so you need to put a comment</p><p> </p><p>also, VERY BAD hard coded global <code>num_feats</code> <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Adanel/MotorizationRate/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>